src/main/java/edu/hawaii/its/api/service/MembershipService.java

```diff
@@ -76,6 +76,13 @@ public MembershipResults membershipResults(String currentUser, String uid) {
         // The disjoint of basis plus include and exclude: (Basis + Include) - Exclude
         List<String> groupingMembershipPaths = disjoint(parentGroupingPaths(basisIncludeExcludePaths),
                 parentGroupingPaths(excludePaths));
+        // A list of all group paths, in which the uhIdentifier is listed (including curated groupings), so we can find the intersection with curated groupings
+        List<String> allGroupPaths = groupingsService.allGroupPaths(uid);
+        // The list of all curated groupings
+        List<String> curatedGroupings = groupingsService.curatedGroupings();
+        // Intersect the two lists and add to our original list of groupingMembershipPaths
+        allGroupPaths.retainAll(curatedGroupings);
```

Comment:
I am wondering if all uhIdentifiers have the same curated groupings? In that case we would not need to perform the intersection at all.

Reply:
If the question is: if a uhUuid is assigned to one curated grouping, can we assume that it is assigned to all of the curated groupings?  Answer to that question would be no.  A curated grouping is one that we know needs to be released via the uhReleasedGrouping attribute for authorization purposes, but does not belong to a grouping owner.  The IAM team is curating it. The term "curated grouping" should be added to our team dictionary.